code/modules/mob/living/simple_animal/vore/otie.dm

```diff
@@ -261,6 +261,31 @@
 			return
 			..()
 
+/mob/living/simple_animal/otie/death()
```

Comment:
It only worked if the mob died while standing. Although the oxy tweak here kind of also eliminates the main cause of their passive deaths.